### ISSUE TITLE
Uncaught Error: Header must provide a Content-Length property.

### ISSUE THREAD

**Issue by alexanderluna** [atom/ide-typescript#124]
I get this error when I work on my Golang files with every key down event. It seems to conflict with a Go package.

**Atom**: 1.30.0 x64
**Electron**: 2.0.5
**OS**: Mac OS X 10.13.6
**Thrown From**: [ide-typescript](https://github.com/atom/ide-typescript) package 0.7.6

### Stack Trace

Uncaught Error: Header must provide a Content-Length property.

```
At /Users/alexander/.atom/packages/ide-typescript/node_modules/vscode-jsonrpc/lib/messageReader.js:200

Error: Header must provide a Content-Length property.
    at StreamMessageReader.onData (/packages/ide-typescript/node_modules/vscode-jsonrpc/lib/messageReader.js:200:27)
    at /packages/ide-typescript/node_modules/vscode-jsonrpc/lib/messageReader.js:185:19)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at Pipe.onread (net.js:594:20)
```

### Commands

```
     -2:59.6.0 core:move-up (input.hidden-input)
     -2:59.1.0 core:backspace (input.hidden-input)
     -2:57.9.0 core:save (input.hidden-input)
     -1:47.4.0 core:move-up (input.hidden-input)
     -1:45.5.0 core:save (input.hidden-input)
  3x -1:44.5.0 core:move-up (input.hidden-input)
     -1:43.3.0 core:backspace (input.hidden-input)
     -1:41.6.0 core:save (input.hidden-input)
  6x -1:40.8.0 core:move-down (input.hidden-input)
     -1:39.7.0 markdown-writer:insert-new-line (input.hidden-input)
     -1:39.7.0 editor:newline (input.hidden-input)
     -1:39.5.0 markdown-writer:insert-new-line (input.hidden-input)
     -1:39.5.0 editor:newline (input.hidden-input)
  7x -1:25.1.0 core:backspace (input.hidden-input)
     -0:22.1.0 core:move-right (input.hidden-input)
  8x -0:20.6.0 core:move-left (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.33.0 
atom-ide-ui 0.13.0 
atom-jest-snippets 2.1.0 
atom-jinja2 0.6.0 
atom-runner 2.7.1 
autocomplete-modules 2.2.0 
autocomplete-sql 0.5.0 
busy-signal 1.4.3 
color-picker 2.3.0 
emmet 2.4.3 
file-icons 2.1.24 
go-plus 5.8.6 
ide-cpp 0.0.3 
ide-css 0.3.0 
ide-go 0.12.3 
ide-html 0.4.2 
ide-json 0.2.1 
ide-python 1.0.0 
ide-typescript 0.7.6 
intentions 1.1.5 
javascript-snippets 1.2.1 
jekyll 2.1.0 
language-babel 2.85.0 
language-javascript-jsx 0.3.7 
language-liquid 0.7.0 
language-markdown 0.26.0 
language-rspec 0.11.0 
language-rust 0.4.12 
linter 2.2.0 
linter-eslint 8.4.1 
linter-ruby 1.2.6 
linter-rust 0.9.0 
linter-ui-default 1.7.1 
markdown-writer 2.9.0 
minimap 4.29.8 
pigments 0.40.2 
platformio-ide-terminal 2.8.4 
rails-snippets 2.19.0 
react-snippets 1.1.1 
script 3.18.1 
seti-syntax 1.2.0 
```

**Comment by alexanderluna** [comment]
1. I disabled `ide-typescript` and I get the same error saying "The error was thrown from the ide-typescript package."

2. I also uninstalled `ide-typescript` and I get the same error again saying: "The error was thrown from the ide-typescript package."

3. After uninstalling `ide-typescript` I disabled and enabled both `ide-go` and `go-plus` and the error stopped appearing. 

4. I installed the `ide-typescript` package again and I get the error back.
